agasc/scripts/mag_estimate_report.py

```diff
@@ -19,7 +20,9 @@ def parser():
                                        ' CxoTime-compatible time stamp.')
     parse.add_argument('--stop', help='Time to stop processing new observations.'
                                       ' CxoTime-compatible time stamp.')
-    parse.add_argument('--out-dir', default=f'./mag_stats_report',
+    parse.add_argument('--input-dir', default='$SKA/data/agasc',
+                       help='Directory containing mag-stats files')
+    parse.add_argument('--output-dir', default=f'$SKA/www/agasc_supplement_reports/suspect',
```

Comment:
This path seems weird. I would expect more like `$SKA/www/ASPECT/agasc/agasc_supplement_reports/suspect`, or something like that. 

Also (though this not set in stone), I would usually define a configurable root directory like `$SKA/www/agasc` and then have the rest `agasc_supplement_reports/suspect` hardcoded.  I.e. only define root directories, not detail directories down lower.